torch/lib/c10d/ProcessGroupMPI.cpp

```diff
@@ -588,6 +588,167 @@ std::shared_ptr<ProcessGroup::Work> ProcessGroupMPI::reduce_scatter(
   throw std::runtime_error("ProcessGroupMPI does not support reduce_scatter");
 }
 
+std::shared_ptr<ProcessGroup::Work> ProcessGroupMPI::alltoall_base(
+    std::vector<at::Tensor>& outputTensors,
+    std::vector<at::Tensor>& inputTensors,
+    std::vector<int>& outputSplitSizes,
+    std::vector<int>& inputSplitSizes,
```

Comment:
Any reason why these are not `const std::vector<T>&`?

Reply:
I just followed how `ProcessGroup::allgather` is written, no other APIs seem to have `const` before `std::vector<T>&`

```
  virtual std::shared_ptr<ProcessGroup::Work> allgather(
      std::vector<std::vector<at::Tensor>>& outputTensors,
      std::vector<at::Tensor>& inputTensors,
      const AllgatherOptions& opts = AllgatherOptions()) = 0;
```